OmniKitUI/PumpManager/OmnipodHUDProvider.swift

```diff
@@ -86,41 +75,12 @@ internal class OmnipodHUDProvider: NSObject, HUDProvider, PodStateObserver {
             reservoirView.update(volume: reservoirVolume, at: lastInsulinMeasurements.validTime, level: reservoirLevel, reservoirAlertState: reservoirAlertState)
         }
     }
-    
-    private func updateFaultDisplay() {
```

Comment:
We're removing a fault indicator here, which is pretty important. I'm not sure if you have the hud element showing fault information yet, but this should be replaced to use the new way we display faults in Tidepool Loop.

Reply:
at first glance, it seems like the fault is just a icon on the battery. I don't see a similar design. This means I'll make a ticket and dig deeper when I would on generic Pump error and state.

Reply:
at first glance, it seems like the fault is just a icon on the battery. I don't see a similar design. This means I'll make a ticket and dig deeper when I would on generic Pump error and state.

Reply:
https://tidepool.atlassian.net/browse/LOOP-1461

Reply:
Yes, that works. It is just an icon on the display; in the new design, I think it would just be an exclamation with either "Fault" or "Pod Fault".

Reply:
OK. thanks for the hint.